### ISSUE TITLE
Some variables are not in the variables.scss

### ISSUE THREAD

**Issue by nikakey** [sslnyx/Instanews#3]
I suggest to put all the variables in variables.scss (from your mixins, for example) just to make them easier to find and for better organization.

**Comment by sslnyx** [comment]
yes, it will be great if i rewrite the variables. it more about the work and fix afterwords mystic....it will be better to spent more time during coding at the beginning..
